src/components/dashboard/Orders.tsx

```diff
@@ -1,93 +1,316 @@
-import * as React from "react";
-import Link from "@mui/material/Link";
-import Table from "@mui/material/Table";
-import TableBody from "@mui/material/TableBody";
-import TableCell from "@mui/material/TableCell";
-import TableHead from "@mui/material/TableHead";
-import TableRow from "@mui/material/TableRow";
-import Title from "./Title";
-
-// Generate Order Data
-function createData(
-    id: number,
-    date: string,
-    name: string,
-    shipTo: string,
-    paymentMethod: string,
-    amount: number,
-) {
-    return { id, date, name, shipTo, paymentMethod, amount };
+import React, { useState, useEffect, ChangeEvent,useContext } from "react";
```

Comment:
There is some inconsistent spacing on this file. Please use [VS Code Format Document] (https://code.visualstudio.com/docs/editor/codebasics#_formatting) on all the files you worked on.

Reply:
Done